server/db/index.js

```diff
@@ -6,4 +6,9 @@ const options = {
 const pgp = require('pg-promise')(options)
 const db = pgp(project.db)
 
+console.log(`DB Host: ${project.db.host}`)
```

Comment:
Good advice, thanks!

Also, I just found out that Heroku won't let us overwrite DATABASE_URL so we will have to refactor to use the complete string rather than an object. I will edit and resubmit.